tls/s2n_server_extensions.c

```diff
@@ -57,10 +57,10 @@ int s2n_server_extensions_send_size(struct s2n_connection *conn)
 
     GUARD_UINT16_AND_INCREMENT(s2n_server_extensions_server_name_send_size(conn), total_size);
     GUARD_UINT16_AND_INCREMENT(s2n_server_extensions_alpn_send_size(conn), total_size);
-    GUARD_UINT16_AND_INCREMENT(s2n_server_renegotiation_info_ext_size(conn), total_size);
+    total_size += s2n_server_renegotiation_info_ext_size(conn);
     GUARD_UINT16_AND_INCREMENT(s2n_kex_server_extension_size(conn->secure.cipher_suite->key_exchange_alg, conn), total_size);
     GUARD_UINT16_AND_INCREMENT(s2n_server_extensions_max_fragment_length_send_size(conn), total_size);
-    GUARD_UINT16_AND_INCREMENT(s2n_server_session_ticket_ext_size(conn), total_size);
+    total_size += s2n_server_session_ticket_ext_size(conn);
```

Comment:
Removing this check feels dangerous, and breaks the syntactic rule that all functions are guarded. Can we use a compiler pragma to tell it that the constant branch condition is allowed in this function?